website/docs/ref/react.md

```diff
@@ -18,26 +18,35 @@ Default rendering component can be set using `defaultComponent` prop in [`I18nPr
 | Prop name   | Type                                      | Description                                    |
 |-------------| ----------------------------------------- |------------------------------------------------|
 | `className` | string                                    | Class name to be added to `<span>` element     |
-| `render`    | *Function(props) -> Element \| Component* | Custom wrapper rendered as function          |
-| `component` | Component, `null`                         | Custom wrapper component to render translation |
+| `render`    | Function(props) -> Element \| `null`   | Custom render callback to render translation        |
+| `component` | Component \| `null`                         | Custom component to render translation |
 
 `className` is used only for built-in components (when *render* is string).
 
-`Function(props)` props returns the translation, an id, and a message.
+When you use the `render` callback, it obtains an object of type `TransRenderProps` as an argument. If you use `component` prop, you will get the same object as props.
 
-`component` is rendered with the `translation` passed in as its child:
+`TransRenderProps` contains
 
-``` jsx
+- `translation`: the translated message
+- `children`: same as `translation` (for compatibility with React components that expect `children` prop)
+- `id`: the message id
```

Comment:
I deliberately decided not to include the `isTranslated` prop because it's undocumented and kinda buggy:

for example, running the RN example, I see two messages where isTranslated true and false, while I'd expect the value to be the same in both cases (not even sure what would be the right one TBH, but they should be the same)

```
{"props": {"children": "order:", "id": "mfCsxa", "isTranslated": false, "message": "order:", "translation": "order:"}}
```

```
{"props": {"children": "1st message", "id": "+MmZxf", "isTranslated": true, "message": "{messageIndex, selectordinal, one {#st message} two {#nd message} few {#rd message} other {#th message}}", "translation": "1st message"}}
```